client/dist/index.html

```diff
@@ -4,6 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css">
```

Comment:
@gigicupp So this change and also    the css file that you pushed up are not related to testing (the title of this pull request). I know we are all still getting the hang of pull requests and how they are supposed to work, so in the future, try to separate these two file changes into a pull request that solely deals with css/front end work.
BUT, that being said, this and your css look find, it looks like you are just doing some basic experimentation to get some design and structure set up so I don't want to be too critical of it yet. Good job on all your progress!

Reply:
@masoncpott will do.